services/avs/mgmt/src/lib.rs

```diff
@@ -1,6 +1,12 @@
-pub mod models;
-pub mod operations;
-
+#[cfg(feature = "2020-03-20")]
+pub mod models_2020_03_20;
+#[cfg(feature = "2020-03-20")]
+pub use models_2020_03_20 as models;
+#[cfg(feature = "2020-03-20")]
+pub mod operations_2020_03_20;
+#[cfg(feature = "2020-03-20")]
+pub use operations_2020_03_20 as operations;
+#[cfg(feature = "2020-03-20")]
```

Comment:
Can we instead have a module `version_2020_03_20` that is conditionally declared on cfg and then everything else doesn't need the cfg modifier. This should be easier to read.

Reply:
Thanks! I knew there was probably a better way. I'm following https://docs.microsoft.com/en-us/python/api/azure-mgmt-storage/azure.mgmt.storage.v2019_06_01.models?view=azure-python and calling the module `v2019_06_01`.
``` rust
#[cfg(feature = "2020-06-01")]
mod v2020_06_01;
#[cfg(feature = "2020-06-01")]
pub use v2020_06_01::{models, operations, API_VERSION};
```